Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -64,17 +64,30 @@ namespace EMStudio
 
     void AnimViewportWidget::SetupCameras()
     {
-        m_rotateCamera = AZStd::make_shared<AzFramework::RotateCameraInput>(EMStudio::ViewportUtil::BuildRotateCameraInputId());
+        m_rotateCamera = AZStd::make_shared<AzFramework::RotateCameraInput>(EMStudio::ViewportUtil::RotateCameraInputChannelId());
 
-        const auto translateCameraInputChannelIds = EMStudio::ViewportUtil::BuildTranslateCameraInputChannelIds();
+        const auto translateCameraInputChannelIds = EMStudio::ViewportUtil::TranslateCameraInputChannelIds();
         m_translateCamera = AZStd::make_shared<AzFramework::TranslateCameraInput>(
             translateCameraInputChannelIds, AzFramework::LookTranslation, AzFramework::TranslatePivotLook);
         m_translateCamera.get()->m_translateSpeedFn = []
         {
             return 3.0f;
         };
 
-        m_orbitDollyScrollCamera = AZStd::make_shared<AzFramework::OrbitDollyScrollCameraInput>();
+        m_dollyScrollCamera = AZStd::make_shared<AzFramework::OrbitDollyScrollCameraInput>();
```

Comment:
update: This should be added to `m_orbitCamera->m_orbitCameras` as we want this scroll behavior when orbiting (moving closer/further from the pivot). I'd then also add `LookScrollTranslationCameraInput` for scroll movement when in the normal free look camera (this will move the actual position of the camera forward).